{"uuid":"c50cf087-8d06-4e14-9c78-0a5a192e7257","historyId":"42b86949bdd13099cd7021ab627c70b6","fullName":"test.Tests01_RegistrationAndLogin_Mobile.test03_loginInvalidPassword","labels":[{"name":"package","value":"test.Tests01_RegistrationAndLogin_Mobile"},{"name":"testClass","value":"test.Tests01_RegistrationAndLogin_Mobile"},{"name":"testMethod","value":"test03_loginInvalidPassword"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"BuymeMobileAutomation_Appium"},{"name":"subSuite","value":"test.Tests01_RegistrationAndLogin_Mobile"},{"name":"host","value":"Avishay_Avraham"},{"name":"thread","value":"24956@Avishay_Avraham.main(3)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"test03_loginInvalidPassword","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. \nHost info: host: 'AVISHAY_AVRAHAM', ip: '10.0.0.5'\nBuild info: version: '4.19.1', revision: 'abe0ee07dc'\nSystem info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '24.0.1'\nDriver info: io.appium.java_client.android.AndroidDriver\nCommand: [null, newSession {capabilities=[Capabilities {appium:appActivity: , appium:appPackage: il.co.mintapp.buyme, appium:autoGrantPermissions: true, appium:automationName: UIAutomator2, appium:deviceName: androidDevice, appium:newCommandTimeout: 720, appium:resetKeyboard: true, appium:udid: RF8R42F61LA, appium:unicodeKeyBoard: true, platformName: ANDROID}]}]\nCapabilities {appium:appActivity: , appium:appPackage: il.co.mintapp.buyme, appium:autoGrantPermissions: true, appium:automationName: UIAutomator2, appium:deviceName: androidDevice, appium:newCommandTimeout: 720, appium:resetKeyboard: true, appium:udid: RF8R42F61LA, appium:unicodeKeyBoard: true, platformName: ANDROID}","trace":"org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. \nHost info: host: 'AVISHAY_AVRAHAM', ip: '10.0.0.5'\nBuild info: version: '4.19.1', revision: 'abe0ee07dc'\nSystem info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '24.0.1'\nDriver info: io.appium.java_client.android.AndroidDriver\nCommand: [null, newSession {capabilities=[Capabilities {appium:appActivity: , appium:appPackage: il.co.mintapp.buyme, appium:autoGrantPermissions: true, appium:automationName: UIAutomator2, appium:deviceName: androidDevice, appium:newCommandTimeout: 720, appium:resetKeyboard: true, appium:udid: RF8R42F61LA, appium:unicodeKeyBoard: true, platformName: ANDROID}]}]\nCapabilities {appium:appActivity: , appium:appPackage: il.co.mintapp.buyme, appium:autoGrantPermissions: true, appium:automationName: UIAutomator2, appium:deviceName: androidDevice, appium:newCommandTimeout: 720, appium:resetKeyboard: true, appium:udid: RF8R42F61LA, appium:unicodeKeyBoard: true, platformName: ANDROID}\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:536)\r\n\tat io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:270)\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:161)\r\n\tat io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:91)\r\n\tat io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:103)\r\n\tat io.appium.java_client.android.AndroidDriver.<init>(AndroidDriver.java:109)\r\n\tat test.BaseTest.uploadApp(BaseTest.java:59)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:565)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:393)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:326)\r\n\tat org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:810)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:578)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1604)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:819)\r\n\tat org.testng.TestRunner.run(TestRunner.java:619)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:336)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1228)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1134)\r\n\tat org.testng.TestNG.run(TestNG.java:1101)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\nCaused by: java.io.UncheckedIOException: java.net.ConnectException\r\n\tat org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:436)\r\n\tat io.appium.java_client.internal.filters.AppiumIdempotencyFilter.lambda$apply$0(AppiumIdempotencyFilter.java:35)\r\n\tat io.appium.java_client.internal.filters.AppiumUserAgentFilter.lambda$apply$0(AppiumUserAgentFilter.java:87)\r\n\tat org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)\r\n\tat org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute(JdkHttpClient.java:352)\r\n\tat org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:89)\r\n\tat org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:75)\r\n\tat org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:61)\r\n\tat io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:176)\r\n\tat io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:237)\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:518)\r\n\t... 35 more\r\nCaused by: java.net.ConnectException\r\n\tat java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1046)\r\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:215)\r\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:268)\r\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$0(PlainHttpConnection.java:226)\r\n\tat java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:980)\r\n\tat java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:957)\r\n\tat java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:556)\r\n\tat java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1819)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1095)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:619)\r\n\tat java.base/java.lang.Thread.run(Thread.java:1447)\r\nCaused by: java.nio.channels.ClosedChannelException\r\n\tat java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:204)\r\n\tat java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:942)\r\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:203)\r\n\t... 9 more\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1749987763714,"stop":1749987763714}